app/models/test_track/remote/assignment_event.rb

```diff
@@ -3,12 +3,10 @@ class TestTrack::Remote::AssignmentEvent
 
   collection_path '/api/v1/assignment_event'
 
-  attributes :visitor_id, :split_name, :unsynced
+  attributes :visitor_id, :split_name, :mixpanel_result, :context
```

Comment:
context is accepted on the other side of the wire (https://github.com/Betterment/test_track/blob/master/app/controllers/api/v1/assignment_events_controller.rb#L12), felt like it was worth including here for clarity

Reply:
also removing the unused `unsynced` attribute

Reply:
are you sure it's unused? Here's a candidate, and there's duck typing all over the place in this code.

https://github.com/Betterment/test_track_rails_client/blob/14b5650c2090635b6821462705af5f6efe51bd49/app/models/test_track/visitor.rb#L56

Reply:
it's used in assignments, but this class is only used for creating an AssignmentEvent, and the api in TestTrack only exposes a `create` endpoint (so this data isn't being read from anywhere that i can see)

Reply:
cool. i just am gunshy about removing things because the coupling is higher than i'd like at this point in the client code.